lua/cfc_time/server/storage_options/mysql.lua

```diff
@@ -67,8 +69,9 @@ function storage:UpdateBatch( batchData )
         )
 
         transaction:addQuery( query )
-        transaction:start()
     end
+
+    transaction:start()
```

Comment:
We were previously doing this all as individual transactions. Moving the setup+start out of the for loop should let us run them as a batch.